src/Codeception/Test/Feature/CodeCoverage.php

```diff
@@ -26,8 +26,20 @@ public function codeCoverageEnd($status, $time)
             return;
         }
 
+        if (method_exists($this, 'getLinesToBeCovered')) {
```

Comment:
Cept files have `pseudo-annotations` http://codeception.com/docs/07-AdvancedUsage#Groups . I think that additional annotations could be implemented in the same way.